elastio-ec2-backup_ecs_stopped_task/readme.md

```diff
@@ -0,0 +1,67 @@
+# Elastio ec2 backup ecs stopped task
```

Comment:
In English, abbreviations must *always* be capitalized, there are no exceptions.

```suggestion
# Elastio EC2 backup ECS stopped task
```